contracts/SnappAuction.sol

```diff
@@ -151,6 +151,29 @@ contract SnappAuction is SnappBase {
         emit StandingSellOrderBatch(currentBatchIndex, accountId, buyTokens, sellTokens, buyAmounts, sellAmounts);
     }
 
+    function placeSellOrder(
```

Comment:
I was hoping this function could just call the new function with a byte array containing the single order encoded (instead of duplicating the logic).

Reply:
Yes, that would have been nice. Too bad this wasn't mentioned. What do you suggest I do now?

Reply:
I tried to mention it in https://github.com/gnosis/dex-contracts/pull/100#discussion_r292090784

> I'd leave the old interface for convenience and just call into the new method from within the old method (creating a batch of 1 item)

We can create a new PR with just that small improvement.

Reply:
Yes very nice!

Reply:
I would argue that this isn't going to work very well with the double application of `onlyRegistered` and the fact that we have to write an entirely different version of "encodeOrder" that make 26 bytes out of the order data.

From my side, this function would be better off dead.

Reply:
I'll take a crack at it if you don't mind.

Reply:
Although this is still far from correct, this is the best I can do in terms of this suggestion
```
    function placeSellOrder(
        uint8 buyToken,
        uint8 sellToken,
        uint96 buyAmount,
        uint96 sellAmount
    ) public onlyRegistered() {
        uint orderInt = uint(buyToken) + (uint(sellToken) << 8) + (uint(sellAmount) << 16) + (uint(buyAmount) << 112);
        bytes memory orderBytes = new bytes(26);
        assembly { mstore(add(orderBytes, 26), orderInt) }
        placeSellOrders(orderBytes);
```

Reply:
Yes, please @fleupold